neo4j/v1/session.py

```diff
@@ -477,6 +478,12 @@ def healthy(self):
         return None if connection.closed else not connection.defunct
 
     def run(self, statement, parameters=None):
+        if self.transaction:
+            raise ProtocolError("Please close the currently open transaction object before running more "
+                                "statements/transactions in the current session.")
```

Comment:
I am fine with that. Same as before. This is taken from the Java Driver since I figured it made sense to have similar error messages for the same errors